library/options.json

```diff
@@ -82,7 +82,18 @@
       },
       "anti_aliasing": {
         "type": "bool",
-        "default_value": "false"
+        "default_value": "false",
+        "deprecated": "use render.effect.antialiasing.enable instead"
+      },
+      "antialiasing": {
```

Comment:
not a big fan of dropping the `_` but I dont have a suggestion

Reply:
I agree but I cannot find a better alternative

Reply:
yup